test/initialization-cases.html

```diff
@@ -40,20 +39,6 @@
   });
 
   Polymer({
-    is: 'on-ready',
```

Comment:
These tests shouldn't be removed since it's no longer necessary to postpone initialization on `attached` (https://github.com/PolymerElements/iron-location/pull/54#discussion_r108565135)